src/content/docs/cypher/data-definition/create-table.md

```diff
@@ -92,4 +92,15 @@ The query above is equivalent to the following:
 MATCH (a:User)-[:Knows_User_User|:Knows_User_city]->(b) RETURN *;
 ```
 
-As you can imagine, the more relationships you want to selectively query on, the more useful relationship table groups become.
\ No newline at end of file
+As you can imagine, the more relationships you want to selectively query on, the more useful relationship table groups become.
+
+## IF NOT EXISTS
+By default, kuzu throws an exception if the given table name already exists in the database. `IF NOT EXISTS` syntax instructs kuzu do nothing when
```

Comment:
```suggestion
If the given table name already exists in the database, Kùzu throws an exception when you try to create it. To avoid the exception being raised, use the `IF NOT EXISTS` clause. This tells Kùzu to do nothing when
```